Syntaxes/Swift.tmLanguage

```diff
@@ -4273,9 +4278,26 @@
 					<key>match</key>
 					<string>(?&lt;!\.)\b(do)\b(\s*)</string>
 				</dict>
+				<dict>
+					<key>captures</key>
+					<dict>
+						<key>1</key>
+						<dict>
+							<key>name</key>
+							<string>keyword.control.async.swift</string>
+						</dict>
+						<key>2</key>
+						<dict>
+							<key>name</key>
+							<string>keyword.other.declaration-specifier.swift</string>
+						</dict>
+					</dict>
+					<key>match</key>
+					<string>(?&lt;!\.)\b(?:(async)\s+)?(let|var)\b</string>
+				</dict>
 				<dict>
 					<key>match</key>
-					<string>(?&lt;!\.)\b(?:associatedtype|let|operator|typealias|var)\b</string>
+					<string>(?&lt;!\.)\b(?:associatedtype|operator|typealias)\b</string>
```

Comment:
Was this just a cleanup? It doesn't seem like it changes anything, so is probably fine.

Reply:
Yep just removed since it would duplicate the other new rule added